analyses/run-gistic/scripts/prepare_seg_for_gistic.R

```diff
@@ -0,0 +1,57 @@
+# Adapted from https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/f02b6fe1198aed281a650b679f79c261cda00ffd/analyses/run-gistic/scripts/prepare_seg_for_gistic.R
+# prepare_seg_for_gistic.R
+#
+# 
+# Purpose: Generate seg files that are compatible with GISTIC 
+# Currently, if we use the cnv consensus seg file as is, since we define copy.num=NA for neutral calls
+# a lot of NAs are present in the file and will cause GISTIC to error out. 
+# Those changes were made for the focal CN module so in order not to change that but also fixed the 
+# issue in GISTIC, we generated a consensus seg file just for GISTIC by using `tumor_ploidy` as `copy.num`
+
+
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Parse command line options
+library(optparse)
+library(tidyverse)
```

Comment:
I know the originating script loads `tidyverse`, but it also uses `::` syntax throughout such that the library load is extraneous. You can remove this line entirely and keep the rest, your call.